docs/set.rst

```diff
@@ -0,0 +1,471 @@
+Sets
+====
+
+:haddock:`unordered-containers`
+
+:haddock:`unordered-containers/Data.HashSet`
+
+:haddock:`unordered-containers/Data.HashSet#empty`
```

Comment:
These links really don't seem to belong here.

Reply:
Oops, these were added when I was testing the autolinker and I forgot to remove them.